docs/lib/examples/Popover.js

```diff
@@ -24,7 +24,7 @@ export default class Example extends React.Component {
         <Button id="Popover1" onClick={this.toggle}>
           Launch Popover
         </Button>
-        <Popover placement="bottom" isOpen={this.state.popoverOpen} target="Popover1" toggle={this.toggle}>
+        <Popover placement="bottom" isOpen={this.state.popoverOpen} closeOnClick={ true } target="Popover1" toggle={this.toggle}>
```

Comment:
It works even without passing a boolean. Edited the example to show this more clearly.